packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1711,6 +1712,40 @@ QUnit.module('options changed', moduleSetup, () => {
     });
 });
 
+QUnit.module('focusOnDeletion', () => {
```

Comment:
so, what is "focusOnDeletion"? :) 
It's just a name of s module, write it as u usually write sentences.
`focus on deleting`

Reply:
[b8bba63](https://github.com/DevExpress/DevExtreme/pull/27230/commits/b8bba636980d2478c625678291fb31d98286ed70)